### ISSUE TITLE
beat.sh script cannot use a custom config path

### ISSUE THREAD

**Issue by tsg** [elastic/beats#2171]
In alpha5 we added the option to use the `-c` multiple times. This somewhat breaks the beat.sh scripts that we put in the `/usr/bin` for DEB/RPM installations (the beat binary is not in $PATH by default). 

Here is an example of a script:

```
#!/bin/bash

# Script to run {.beat_name} in foreground with the same path settings that
# the init script / systemd unit file would do.

/usr/share/filebeat/bin/filebeat -e \
  -c /etc/filebeat/filebeat.yml \
  -path.home /usr/share/filebeat \
  -path.config /etc/filebeat \
  -path.data /var/lib/filebeat \
  $@
```

One of the things that was possible in alpha4 was to specify a custom config file, e.g.:

```
filebeat.sh -e -c filebeat.yml
```

This was overriding the default config file from the script with `filebeat.yml`. With alpha5, however, the Beat combines the two configuration files, which comes as a surprise for the user.

**Comment by tsg** [comment]
IMO, the best solution would be if the default value for `-c` is `filebeat.yml` which is searched in `path.config`. Then we don't need to put it explicitly in the script, and the user can override it with `-c` passed to the script.

Have to check if this is a viable solution, I know we had issues with this approach before, but I think that was before introducing the config paths.

**Comment by urso** [comment]
I think by default (if no -c is given), filebeat should load config file from ${path.config}/filebeat.yml by default. Solution might be to remove `-c` from script altogether. If removing `-c` doesn’t work properly we indeed have a bug in filebeat itself.

**Comment by tsg** [comment]
Fixed by #2245.
